packages/yoroi-extension/app/components/wallet/staking/dashboard-revamp/OverviewDialog.js

```diff
@@ -84,7 +88,26 @@ export default class OverviewModal extends Component<Props> {
         <Box display="flex" alignItems="center" py="50px">
           <Box display="flex" alignItems="center" flex="1">
             <Typography mr="6px">{intl.formatMessage(messages.availableTotalRewards)}</Typography>
-            <HelperTooltip message={intl.formatMessage(messages.availableTotalRewardsHelper)} />
+            <HelperTooltip
+              message={
+                <p>
+                  <span>{intl.formatMessage(messages.availableTotalRewardsHelper)}</span>
+                  <Link
```

Comment:
Please do not concatenate parts into a sentence like this, because not all natural languages have the same grammatical structure like English.

Reply:
Use `<FormattedMessage ... />` instead.